test_schemas/en/test_list_summary_on_question.json

```diff
@@ -369,6 +369,43 @@
                                     }
                                 }
                             }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "test-number-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "test-currency",
+                                        "description": "",
+                                        "label": "Currency Test",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "GBP"
+                                    },
+                                    {
+                                        "id": "test-square-kilometres",
+                                        "description": "",
+                                        "label": "Kilometres Square",
+                                        "mandatory": false,
+                                        "type": "Unit",
+                                        "unit_length": "short",
+                                        "unit": "area-square-kilometer"
+                                    },
+                                    {
+                                        "id": "test-decimal",
+                                        "description": "",
+                                        "label": "Decimal test to 2 decimal places",
+                                        "mandatory": false,
+                                        "type": "Number",
+                                        "decimal_places": 2
+                                    }
+                                ],
+                                "description": "",
+                                "id": "test-number-range-question",
+                                "title": "Please enter test values (non-mandatory)",
+                                "type": "General"
+                            }
```

Comment:
Is this additional question needed given there is already a non-list question "Are any of these people related to one another"?

Reply:
No :+1: